### PR TITLE
Fix a bug in LocalcacheFileSystem

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: core/client/hdfs/src/main/java/alluxio/hadoop/AlluxioHdfsFileSystem.java [hunk 1]
```diff
@@ -119,7 +138,7 @@ public boolean exists(AlluxioURI alluxioURI, ExistsPOptions options) {
 
   @Override
   public AlluxioConfiguration getConf() {
-    return InstancedConfiguration.defaults();
+    return mAlluxioConf;
```
Comment (apc999) [inline core/client/hdfs/src/main/java/alluxio/hadoop/AlluxioHdfsFileSystem.java:141]: we shall not return the default configuration but the real Alluxio config as `LocalCacheFileInStream` depends on this https://github.com/Alluxio/alluxio/blob/e488f4d48d6415e664203d5d3fc84e755ff328b8/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java#L102

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.2.2 please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**